docs/vsphere_ci/README.md

```diff
@@ -99,6 +100,16 @@ To enable detailed logging:
   PACKER_LOG=1 packer build build.json
 ```
 
+### What to do during the Packer build
+
+During the golden image creation, it is highly recommended to establish access to the virtual machine by launching a
+Web Console through the vSphere Client. This can be done after the Packer build has powered on the VM (while it is
```

Comment:
I think this might be more correct?
vSphere Client -> vCenter console

Reply:
I'm referring to the web page itself, which is a component of a vCenter server

> With vCenter Server, you get access to the new HTML 5 Client

Will change to `vCenter web client` for clarity